packages/suspense/src/cache/createCache.test.ts

```diff
@@ -647,7 +647,7 @@ describe("createCache", () => {
       gcCache.cache({ key: "test" }, "test");
       expect(gcCache.getValueIfCached("test")).toEqual({ key: "test" });
 
-      expect(weakRefArray.length).toBe(1);
+      expect(weakRefArray.length).toBe(2);
```

Comment:
I don't understand this commit. It looks kind of like it's "fixing" the test by changing the expectations to be values that wouldn't be expected? Am I misreading it?

Reply:
Oh! Haha, no need for apologies. I just added those two together and misinterpreted the situation 😁 

Okay, that makes sense, although the tests now look a bit odd at a glance:
```js
expect(weakRefArray.length).toBe(2); // Not obvious at a glance why 2
weakRefArray[0].collect(); // Not obvious why we only collect/evict 1; shouldn't we evict all?
```

I wonder if there's a way we can write them to be clearer? 😄